src/pages/dao/index.jsx

```diff
@@ -113,16 +113,16 @@ export const DAO = () => {
               governance purposes within the Teia DAO ecosystem. TEIA tokens do
               not represent any form of investment in the Teia DAO or any
               associated entity. TEIA tokens are not intended to be securities
-              or investment assets. Teia does not intend to generate and
+              or investment assets. The Teia DAO LLC does not intend to generate and
               distribute profits among its members. Teia does not sell TEIA
               tokens, nor to generate profit with the tokens. Claiming TEIA
               tokens via this claim page is free.
             </li>
 
             <li counter="2">
               <b>No investment advice:</b> The information provided on the claim
-              page, including but not limited to any articles, blog posts on
-              blog.teia.art, or other related materials, is for informational
+              page bor any articles, blog posts on blog.teia.art, or other related materials
```

Comment:
Typo here @merchantcoppola but not sure what it should be, maybe "for" but it would be strange in context?

Reply:
thx, i did a small update for that, do you think its better now?

Reply:
Yes thanks!